prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -114,6 +114,325 @@ static int tcpx_ep_msg_xfer_enable(struct tcpx_ep *ep)
 	return ret;
 }
 
+static int proc_conn_resp(struct poll_fd_info *poll_info,
+			  struct tcpx_ep *ep)
+{
+	struct ofi_ctrl_hdr conn_resp;
+	struct fi_eq_cm_entry *cm_entry;
+	ssize_t len;
+	int ret = FI_SUCCESS;
+
+	ret = rx_cm_data(ep->conn_fd, &conn_resp, ofi_ctrl_connresp, poll_info);
+	if (ret)
+		return ret;
+
+	cm_entry = calloc(1, sizeof(*cm_entry) + poll_info->cm_data_sz);
+	if (!cm_entry)
+		return -FI_ENOMEM;
+
+	cm_entry->fid = poll_info->fid;
+	memcpy(cm_entry->data, poll_info->cm_data, poll_info->cm_data_sz);
+
+	ret = tcpx_ep_msg_xfer_enable(ep);
+	if (ret)
+		goto err;
+
+	len = fi_eq_write(&ep->util_ep.eq->eq_fid, FI_CONNECTED, cm_entry,
+			  sizeof(*cm_entry) + poll_info->cm_data_sz, 0);
+	if (len < 0) {
+		ret = (int) len;
+		goto err;
+	}
+err:
+	free(cm_entry);
+	return ret;
+}
+
+int tcpx_eq_wait_try_func(void *arg)
+{
+	return FI_SUCCESS;
+}
+
+static void client_wait_for_connresp(struct util_wait *wait,
+				     struct poll_fd_info *poll_info)
```

Comment:
AFAICT, this call doesn't actually wait for a response.  If that's the case, please rename the call.  If the call is actually blocking until the server sends a response, then we have other issues and need to change that behavior. :)